src/core/config.py

```
@@ -2,44 +2,53 @@
 from pydantic import BaseModel, Field
 
 
+class AudioProcessorConfig(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)